src/pages/docs/liveobjects/map.mdx

```diff
@@ -147,6 +149,8 @@ Example structure of an update object when the key `foo` is updated and the key
 ```
 </Code>
 
+Note that subscribing to data updates on a `LiveMap` does not affect the number of billable messages received by a client. Any client attached to a channel with the `object-subscribe` capability automatically receives all object messages for that channel. Subscribing to updates on an object simply allows the client to react to changes in that object.
```

Comment:
> Subscribing to updates on an object simply allows the client to react to changes in that object.
I think this is confusing.
"allows" suggests something to do with permission - I'd propose "enables".

I think the clearest way to say this is
> Subscribing to updates on an object simply adds a listener that is called whenever updates for that object are received by the client.